app/workers/deletes.php

```diff
@@ -10,6 +10,9 @@
 use Utopia\Abuse\Adapters\TimeLimit;
 use Utopia\CLI\Console;
 use Utopia\Audit\Audit;
+use Utopia\Cache\Adapter\Redis as RedisCache;
+use Utopia\Cache\Cache;
+use Utopia\Database\Adapter\MariaDB;
```

Comment:
Since database creation happens in the parent worker script, do we still need these imports?
```suggestion
```